src/lib/pages/validator-details/components/performance/index.tsx

```diff
@@ -11,24 +16,40 @@ interface PerformanceProps {
   validatorAddress: ValidatorAddr;
 }
 
-export const Performance = ({ validatorAddress }: PerformanceProps) => (
-  <Flex direction="column" gap={{ base: 4, md: 6 }} pt={6}>
-    <Flex gap={{ base: 4, md: 6 }} direction={{ base: "column", md: "row" }}>
-      <Flex flex={{ md: "2" }}>
-        <UptimeSection isDetailPage />
+export const Performance = ({ validatorAddress }: PerformanceProps) => {
```

Comment:
can we add `onViewMore` field to its props and use this on the overview section? then in `UptimeSection` component, `uptimeData` shouldn't be `undefined`